test/squish/deadline_gui_test_samples/job_bundle_one/template.yaml

```diff
@@ -0,0 +1,108 @@
+specificationVersion: 'jobtemplate-2023-09'
+name: GUI Submitter Testing - Job Bundle One
+description: |
+  This is a custom job bundle created for the Squish Automated Test Suite which verifies the GUI Submitter UI. 
+  It is one of two custom job bundles created for the Squish Automated Test Suite. 
+  This is not intended for general use other than testing the GUI Submitter UI in Deadline Cloud CLI.
+
+parameterDefinitions:
+# Input Details
+- name: MockJobName
+  type: STRING
+  userInterface:
+    control: LINE_EDIT
+    label: Mock Job Name
+    groupLabel: Mock Job Details
+  default: Squish GUI Submitter Test - Job Bundle One
+  description: Enter the name of the job.
+- name: Frames
+  type: STRING
+  userInterface:
+    control: LINE_EDIT
+    groupLabel: Mock Job Details
+  default: 1-150
+  description: Enter the number of frames to test.
+- name: SquishMockSceneFile
+  type: PATH
+  objectType: FILE
+  dataFlow: IN
+  userInterface:
+    control: CHOOSE_INPUT_FILE
+    label: GUI Submitter Test File
+    groupLabel: Mock Job Details
+  default: "./mock_test_scene/squish_anim_test.txt"
+  description: Choose the GUI Submitter file to run tests against.
+# Checkboxes
+- name: CheckBoxOne
+  type: STRING
+  userInterface:
+    groupLabel: Custom Plugins
+    control: CHECK_BOX
+    label: Squish Custom Plugin 
+  default: "True"
+  allowedValues: ["True", "False"]
+  description: This checkbox defaults to "True".  
+- name: CheckBoxTwo
+  type: STRING
+  userInterface:
+    groupLabel: Custom Plugins
+    control: CHECK_BOX
+    label: Additional Squish Custom Plugin
+  default: "False"
+  allowedValues: ["True", "False"]
+  description: This checkbox defaults to "False".  
+# Output Details
+- name: OutputDir
+  type: PATH
+  objectType: DIRECTORY
+  dataFlow: OUT
+  userInterface:
+    control: CHOOSE_DIRECTORY
+    label: Output Directory
+    groupLabel: Output Details
+  default: "./output_images"
+  description: Choose the GUI Submitter test output directory.
+- name: OutputPattern
+  type: STRING
+  userInterface:
+    control: LINE_EDIT
+    label: Output File Pattern
+    groupLabel: Output Details
+  default: "gui_submitter_output_####"
+  description: Enter the output filename pattern (without extension).
+- name: Format
+  type: STRING
+  userInterface:
+    control: DROPDOWN_LIST
+    label: Output File Format
+    groupLabel: Output Details
+  description: Choose the file format to render as.
+  default: OPEN_EXR
+  allowedValues: [TGA, RAWTGA, JPEG, IRIS, IRIZ, PNG, HDR, TIFF, OPEN_EXR, OPEN_EXR_MULTILAYER, CINEON, DPX, DDS, JP2, WEBP]
+# Steps
+steps:
+- name: GUISubmitterTesting
+  script:
+    actions:
+      onRun:
+        command: bash
+        args: ['{{Task.File.runScript}}']
+    embeddedFiles:
+      - name: runScript
+        type: TEXT
+        data: |
+          #!/usr/bin/env bash
```

Comment:
Should the embedded script have a `set -euo`?

Reply:
I don't it's necessary given how simple my script is ('echo' is a very simple command (and almost never fails). 

Maybe the only reason to add it is to add consistency across all the Deadline Cloud Job Bundles that have been written, but I noticed some do NOT have `set -euo` added, including GUI Control Showcase which I've referenced: https://github.com/aws-deadline/deadline-cloud-samples/blob/mainline/job_bundles/gui_control_showcase/template.yaml

So I think for now I will exclude it!